tools/github-event-processor/Azure.Sdk.Tools.GitHubEventProcessor/EventProcessing/ScheduledEventProcessing.cs

```diff
@@ -89,6 +94,7 @@ public static async Task CloseAddressedIssues(GitHubEventClient gitHubEventClien
         {
             if (gitHubEventClient.RulesConfiguration.RuleEnabled(RulesConstants.CloseAddressedIssues))
             {
+                int ScheduledTaskUpdateLimit = await gitHubEventClient.ComputeScheduledTaskUpdateLimit();
```

Comment:
Are we logging this value anywhere? If not I'd suggest we do.

Reply:
@weshaggard I'll log this in the ComputeScheduledTaskUpdateLimit function.